salt/cloud/clouds/gce.py

```diff
@@ -400,6 +400,24 @@ def __get_tags(vm_):
     return tags
 
 
+def __get_labels(vm_):
```

Comment:
Since this function takes a param it should be stated in the docstring what this param is.

Reply:
It doesn't need to for a private function.

Plus the vm_ is standard in salt-cloud